src/k2/httpproxy/HTTPProxy.h

```diff
@@ -56,17 +56,11 @@ class HTTPProxy {
     void _registerMetrics();
 
     sm::metric_groups _metric_groups;
-    k2::ExponentialHistogram _readLatency;
-    k2::ExponentialHistogram _writeLatency;
-    k2::ExponentialHistogram _txnLatency;
-    k2::ExponentialHistogram _endLatency;
-
     uint64_t _totalTxns=0;
```

Comment:
We can also remove the totalTxns/reads/writes from the proxy, they can be derived from the histograms you added to the client.

Also let's add deserialization errors, open txn handles, and open query handles as metrics to the proxy.